packages/app/src/app/pages/common/Modals/ShareModal/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { inject, observer } from 'app/componentConnectors';
-import Files from 'embed/components/Files';
+import Files from 'embed/components/legacy/Files';
```

Comment:
🔖 Moved old File tree to legacy because the share modal uses this. Would be gone after the modal's redesign